soc/riscv/riscv-ite/common/chip_chipregs.h

```diff
@@ -863,6 +863,8 @@ struct wdt_it8xxx2_regs {
 
 /* External Timer register fields */
 /* External Timer 3~8 control */
+#define IT8XXX2_EXT_ETX_COMB_RST_EN	(BIT(3) | BIT(1) | BIT(0))
```

Comment:
Nit - define this using the `ETXCOMB`, `EXTRST`, `EXTEN` definitions.

Reply:
Done